<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SearchDialog</class>
 <widget class="QWidget" name="SearchDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>594</width>
    <height>206</height>
   </rect>
  </property>
  <layout class="QGridLayout" name="topLayout" rowstretch="0,0,0,1" columnstretch="0,0,1,0,0,0">
   <property name="margin">
    <number>0</number>
   </property>
   <item row="1" column="0">
    <widget class="QToolButton" name="displayOptions">
     <property name="text">
      <string>...</string>
     </property>
     <property name="checkable">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="1" column="4">
    <widget class="KPushButton" name="replaceButton">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="text">
      <string>Replace</string>
     </property>
    </widget>
   </item>
   <item row="0" column="4">
    <widget class="KPushButton" name="searchButton">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="text">
      <string>Search</string>
     </property>
    </widget>
   </item>
   <item row="0" column="0">
    <widget class="QToolButton" name="newTabButton">
     <property name="text">
      <string>...</string>
     </property>
    </widget>
   </item>
   <item row="0" column="1" alignment="Qt::AlignRight">
    <widget class="QLabel" name="findLabel">
     <property name="text">
      <string>Find</string>
     </property>
     <property name="buddy">
      <cstring>searchCombo</cstring>
     </property>
    </widget>
   </item>
   <item row="1" column="1" alignment="Qt::AlignRight">
    <widget class="QLabel" name="replaceLabel">
     <property name="text">
      <string>Replace</string>
     </property>
     <property name="buddy">
      <cstring>replaceCombo</cstring>
     </property>
    </widget>
   </item>
   <item row="3" column="0" colspan="6">
    <widget class="QStackedWidget" name="stackedWidget">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Minimum">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="currentIndex">
      <number>1</number>
     </property>
     <widget class="QWidget" name="resultPage">
      <layout class="QVBoxLayout" name="verticalLayout">
       <property name="margin">
        <number>0</number>
       </property>
       <item>
        <widget class="KTabWidget" name="resultTabWidget">
         <property name="tabsClosable">
          <bool>true</bool>
         </property>
         <property name="movable">
          <bool>true</bool>
         </property>
         <property name="automaticResizeTabs">
          <bool>true</bool>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="optionsPage">
      <layout class="QGridLayout" name="optionsLayout">
       <property name="margin">
        <number>0</number>
       </property>
       <item row="2" column="0">
        <spacer name="verticalSpacer">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>1</width>
           <height>1</height>
          </size>
         </property>
        </spacer>
       </item>
       <item row="0" column="0">
        <widget class="QWidget" name="folderOptions" native="true">
         <property name="sizePolicy">
          <sizepolicy hsizetype="MinimumExpanding" vsizetype="Minimum">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <layout class="QGridLayout" name="gridLayout_2" rowstretch="0,0,0,0,0,0" columnstretch="0,0,0,0,0,10">
          <property name="margin">
           <number>0</number>
          </property>
          <item row="1" column="2">
           <widget class="QCheckBox" name="useRegExp">
            <property name="text">
             <string>Regular e&amp;xpressions</string>
            </property>
           </widget>
          </item>
          <item row="0" column="0" colspan="6">
           <widget class="Line" name="line">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
           </widget>
          </item>
          <item row="2" column="1" colspan="5">
           <layout class="QHBoxLayout" name="horizontalLayout_3">
            <item>
             <widget class="KUrlComboRequester" name="folderRequester">
              <property name="mode">
               <set>KFile::Directory|KFile::ExistingOnly|KFile::LocalOnly</set>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QToolButton" name="folderUpButton">
              <property name="toolTip">
               <string>Go one folder up.</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QToolButton" name="currentFolderButton">
              <property name="toolTip">
               <string>Use the current document's path.</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item row="3" column="1" colspan="5">
           <widget class="KComboBox" name="filterCombo">
            <property name="editable">
             <bool>true</bool>
            </property>
            <property name="insertPolicy">
             <enum>QComboBox::InsertAtTop</enum>
            </property>
            <item>
             <property name="text">
              <string>*</string>
             </property>
            </item>
           </widget>
          </item>
          <item row="3" column="0">
           <widget class="QLabel" name="filterLabel">
            <property name="text">
             <string>Filter</string>
            </property>
            <property name="alignment">
             <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
            </property>
            <property name="buddy">
             <cstring>filterCombo</cstring>
            </property>
           </widget>
          </item>
          <item row="1" column="3">
           <widget class="QCheckBox" name="expandResults">
            <property name="text">
             <string extracomment="When this is checked the result treeview will be expanded after a search">Expand results</string>
            </property>
           </widget>
          </item>
          <item row="5" column="1">
           <widget class="QCheckBox" name="recursiveCheckBox">
            <property name="text">
             <string>Recursive</string>
            </property>
            <property name="checked">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item row="4" column="1" colspan="5">
           <widget class="KComboBox" name="excludeCombo">
            <property name="editable">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item row="2" column="0">
           <widget class="QLabel" name="folderLabel">
            <property name="text">
             <string>F&amp;older</string>
            </property>
            <property name="alignment">
             <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
            </property>
            <property name="buddy">
             <cstring>folderRequester</cstring>
            </property>
           </widget>
          </item>
          <item row="4" column="0" alignment="Qt::AlignRight">
           <widget class="QLabel" name="excludeLabel">
            <property name="text">
             <string>Exclude</string>
            </property>
            <property name="buddy">
             <cstring>excludeCombo</cstring>
            </property>
           </widget>
          </item>
          <item row="1" column="1">
           <widget class="QCheckBox" name="matchCase">
            <property name="text">
             <string>&amp;Match case</string>
            </property>
           </widget>
          </item>
          <item row="5" column="4">
           <widget class="QCheckBox" name="binaryCheckBox">
            <property name="text">
             <string>Include binary files</string>
            </property>
           </widget>
          </item>
          <item row="5" column="2">
           <widget class="QCheckBox" name="hiddenCheckBox">
            <property name="text">
             <string>Include hidden</string>
            </property>
           </widget>
          </item>
          <item row="5" column="3">
           <widget class="QCheckBox" name="symLinkCheckBox">
            <property name="text">
             <string>Follow symbolic links</string>
            </property>
           </widget>
          </item>
          <item row="1" column="4" colspan="2">
           <layout class="QHBoxLayout" name="horizontalLayout_4">
            <item>
             <spacer name="horizontalSpacer">
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>1</width>
                <height>1</height>
               </size>
              </property>
             </spacer>
            </item>
            <item>
             <widget class="QLabel" name="label">
              <property name="text">
               <string>Se&amp;arch:</string>
              </property>
              <property name="buddy">
               <cstring>searchPlaceCombo</cstring>
              </property>
             </widget>
            </item>
            <item>
             <widget class="KComboBox" name="searchPlaceCombo">
              <item>
               <property name="text">
                <string>in Open files</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>in Folder</string>
               </property>
              </item>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
   <item row="0" column="5">
    <widget class="QStackedWidget" name="nextAndStop">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="page_3">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <property name="margin">
        <number>0</number>
       </property>
       <item>
        <widget class="KPushButton" name="nextButton">
         <property name="enabled">
          <bool>false</bool>
         </property>
         <property name="text">
          <string>Next</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="stopButtonPage">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <property name="margin">
        <number>0</number>
       </property>
       <item>
        <widget class="KPushButton" name="stopButton">
         <property name="sizePolicy">
          <sizepolicy hsizetype="MinimumExpanding" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="text">
          <string>Stop</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
   <item row="0" column="2" colspan="2">
    <widget class="KHistoryComboBox" name="searchCombo">
     <property name="editable">
      <bool>true</bool>
     </property>
     <property name="autoCompletion">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="1" column="2" colspan="2">
    <widget class="KHistoryComboBox" name="replaceCombo">
     <property name="editable">
      <bool>true</bool>
     </property>
     <property name="autoCompletion">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="1" column="5">
    <widget class="KPushButton" name="replaceCheckedBtn">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="text">
      <string>Replace checked</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>KComboBox</class>
   <extends>QComboBox</extends>
   <header>kcombobox.h</header>
  </customwidget>
  <customwidget>
   <class>KUrlComboRequester</class>
   <extends>KUrlRequester</extends>
   <header>kurlrequester.h</header>
  </customwidget>
  <customwidget>
   <class>KPushButton</class>
   <extends>QPushButton</extends>
   <header>kpushbutton.h</header>
  </customwidget>
  <customwidget>
   <class>KUrlRequester</class>
   <extends>QFrame</extends>
   <header>kurlrequester.h</header>
  </customwidget>
  <customwidget>
   <class>KTabWidget</class>
   <extends>QTabWidget</extends>
   <header>ktabwidget.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>KHistoryComboBox</class>
   <extends>KComboBox</extends>
   <header>khistorycombobox.h</header>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>newTabButton</tabstop>
  <tabstop>searchCombo</tabstop>
  <tabstop>replaceCombo</tabstop>
  <tabstop>searchButton</tabstop>
  <tabstop>nextButton</tabstop>
  <tabstop>stopButton</tabstop>
  <tabstop>replaceButton</tabstop>
  <tabstop>replaceCheckedBtn</tabstop>
  <tabstop>matchCase</tabstop>
  <tabstop>useRegExp</tabstop>
  <tabstop>expandResults</tabstop>
  <tabstop>searchPlaceCombo</tabstop>
  <tabstop>folderRequester</tabstop>
  <tabstop>folderUpButton</tabstop>
  <tabstop>currentFolderButton</tabstop>
  <tabstop>filterCombo</tabstop>
  <tabstop>excludeCombo</tabstop>
  <tabstop>recursiveCheckBox</tabstop>
  <tabstop>hiddenCheckBox</tabstop>
  <tabstop>symLinkCheckBox</tabstop>
  <tabstop>binaryCheckBox</tabstop>
  <tabstop>displayOptions</tabstop>
  <tabstop>resultTabWidget</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
